Prace_domowe/Praca_domowa3/Grupa1/Kacper_Grzymkowski/PD3.ipynb

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
<p>SVM rzeczywiście często długo się trenuje. To czy zbiegnie do rozwiązania bardzo zależy od doboru parametrów (m.in. wartość C) - temu zagadnieniu poświęcona jest kolejna praca domowa. Tak jak jest napisane w warningu - w przypadku SVM warto wcześniej przeskalować dane.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>